AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -464,8 +470,8 @@ - (void)setAsyncDataSource:(id<ASCollectionDataSource>)asyncDataSource
   if (asyncDataSource == nil) {
     _asyncDataSource = nil;
     _proxyDataSource = _isDeallocating ? nil : [[ASCollectionViewProxy alloc] initWithTarget:nil interceptor:self];
-    
-    memset(&_asyncDataSourceFlags, 0, sizeof(_asyncDataSourceFlags));
+    _asyncDataSourceFlags = {};
+
```

Comment:
Should not have space after.  I'm split on this change -- I think the memset is much more explicit, at least since I'm not sure it is widespread knowledge that {} will zero memory for the specific size of a struct type (particularly in an equality like this, where we need to overwrite existing values, as opposed to initialization-time).